src/main/resources/META-INF/plugin.xml

```diff
@@ -411,6 +414,15 @@
                             description="A CDI bean annotated with `@Named` can be referenced in any template through `cdi` and/or `inject` namespaces."
                             url="https://quarkus.io/guides/qute-reference#injecting-beans-directly-in-templates"
                             implementationClass="com.redhat.devtools.intellij.qute.psi.internal.extensions.quarkus.InjectNamespaceResolverSupport"/>
+
+    <!-- Data model providers for Quarkus Renarde -->
+    <qute.dataModelProvider namespaces="uri,uriabs"
+                            description="If you want a URI to an endpoint in a Qute view, you can use the `uri` and `uriabs` namespace with a call to the endpoint you want to point to."
```

Comment:
In order to generate the URI to an endpoint, in a Qute view, ...

Reply:
fixed